tests/test_install.ts

```diff
@@ -187,6 +188,34 @@ describe("Agama test", function () {
 
     // For development will be useful to stop before starting installation
     if (agamaInstall === true) {
+        it("should create first user", async function (){
```

Comment:
this test should outside the block for installing, you can exercise it without installing the system for real.

Reply:
Solved bbf69efe39e06cc95d295f663d3b220933fc32d0